tests/bdd/ha-bhyve02/test_NAS_T0961.py

```diff
@@ -163,32 +146,30 @@ def navigate_to_system_setting_and_click_misc(driver):
 def the_miscellaneous_page_should_open(driver):
     """the Advanced page should open."""
     assert wait_on_element(driver, 7, xpaths.advanced.title)
-    assert wait_on_element(driver, 7, xpaths.advanced.systemDatasetPool_card)
+    assert wait_on_element(driver, 7, xpaths.advanced.system_datasetPool_card)
```

Comment:
Should system_**datasetPool**_card be Snake_Case ?  ( system_Dataset_Pool_card )